ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -336,14 +336,21 @@ export default class ConfirmApproveContent extends Component {
             : t('functionApprove')}
         </div>
         {isSetApproveForAll && isApprovalOrRejection !== undefined ? (
-          <div className="confirm-approve-content__small-text">
-            {`${t('parameters')}: ${isApprovalOrRejection}`}
+          <>
+            <div className="confirm-approve-content__small-text">
```

Comment:
Should this also be a `Text` usage?  Or `Box`?

Reply:
@darkwing My idea was to use Text since it will translate to a <p> tag, and inside is text, doesn't Box translate to a <div> tag? Therefore making semantically more sense to use the <Text>?

Reply:
rofl, forget it, I misunderstood what you were referring to. Yes the <div> can be a Box, 100% 👍🏼